### ISSUE TITLE
Source code for the .NET Framework version is throwing an error when executing a build.

### ISSUE THREAD

**Issue by tmosson1** [kfirprods/NonInvasiveKeyboardHook#7]
Hello,

I just wanted to let you know that I am getting the following error below when I trying to execute a build of the NonInvasiveKeyboardHookLibrary .NET Framework version.

**Your project does not reference ".NETFramework,Version=v4.5.2" framework. Add a reference to ".NETFramework,Version=v4.5.2" in the "TargetFrameworks" property of your project file and then re-run NuGet restore.**	

I checked the properties and I also unloaded the project file to check the XML markup.  It is set to .NET Framework version 4.5.2.

Please follow-up on this issue whenever you get a chance to do so.

Thank you.

**Comment by tmosson1** [comment]
Also I cannot install the nuget package on my .NET Framework 4.8 winform application project as it also throws an error.  I didn't capture the error message but it was pointing out that it conflicted with .NET Framework 4.8.

**Comment by tmosson1** [comment]
I think I got it to build.  I had to create my own project and add the existing source code over and build it there.  I was able to build it with .NET Framework 4.8.

**Comment by kfirprods** [comment]
> I think I got it to build. I had to create my own project and add the existing source code over and build it there. I was able to build it with .NET Framework 4.8.

Thank you for this information. I will try to change the package's configuration to make it compatible with .NET framework 4.8, and I'll post here with conclusions soon :)

**Comment by 0xotHik** [comment]
Yep, can confirm — NuGet package was not installing not on .Net Framework 4.6 project, nor on 4.7.2

"NonInvasiveKeyboardHookLibrary" project from 2.1.1 source code was not building either. Created a new project, added *.cs files — built successfully.

(But it's a trifle; thank you for your work on the library 🤓)

**Comment by Dellgenius** [comment]
Had the same issue, was getting that error when trying to install 2.11.

However, when I installed the 2.1.0 nuget package, it installed just fine.

**Comment by 0xotHik** [comment]
Okay, NuGet 2.1.1 successfully installed after I migrate my project from .Net Framework 4.7.2 to .Net 5

**Comment by gbadwaik2706** [comment]
Deleting Bin and Obj folder worked for me.
